modules/claims_api/app/controllers/claims_api/v0/forms/power_of_attorney_controller.rb

```diff
@@ -79,6 +79,8 @@ def status
         def active
           power_of_attorney = ClaimsApi::PowerOfAttorney.find_using_identifier_and_source(header_md5: header_md5,
                                                                                           source_name: source_name)
+
+          validate_poa_code_for_current_user!(current_poa_code)
```

Comment:
i might be misunderstanding but im not sure this is what we need, i think something more like
```
# method in concerns/poa_verfication
# https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/concerns/claims_api/poa_verification.rb#L30
def validate_user_is_accredited! 
    representative = ::Veteran::Service::Representative.for_user(first_name: @current_user.first_name,
                                                                                                     last_name: @current_user.last_name)
    raise ::Common::Exceptions::Forbidden if representative.blank?
end


validate_user_is_accredited! if header_request?
```

what is currently here, i think, only allows the current representative of a Veteran to use this endpoint. what is described in the ticket is to allow any accredited representative to use this endpoint for any Veteran.

Reply:
ahh ok I see now. I thing I was mixed up on the semantics of the `for_user` method and _who_ the check was actually being applied to